main.js

```diff
@@ -0,0 +1,147 @@
+var city = {
+  name: '',
+  lat: 0,
+  long: 0,
+};
+
+var findWeather = function () { 
+  city.name = $('#city-search').val();
+  $('#city-search').val('');
+  fetchToday(city.name);
+  fetchWeek(city.name);
+  $('.default').removeClass('hide');
+}
+
+$('.search').on('click', function () {
+  findWeather();
+});
+
+var fetchToday = function () {
+  $.ajax({
+    method: "GET",
+    url: "https://api.openweathermap.org/data/2.5/weather?q=" + city.name + "&units=imperial&appid=" + config.apiKey,
+    dataType: "json",
+    success: function(data) {
+      addTodaysWeather(data);
+    },
+    error: function(jqXHR, textStatus, errorThrown) {
+      console.log(textStatus);
+    }
+  })
+}
+var addTodaysWeather = function (data) {
+  city.lat = data.coord.lat;
+  city.long = data.coord.lon;
+  var todaysWeather = {
+    temp: Math.round(data.main.temp),
+    city: data.name,
+    sky: data.weather[0].main,
+    imgURL: data.weather[0].icon
+  }
+  var renderWeather = function () {
+    $('.today').empty();
+  
+    var template = $('#today-template').html();
+    var compiled = Handlebars.compile(template);
+    var newHTML = compiled(todaysWeather);
+    $('.today').append(newHTML);
+  }
+  renderWeather();
+}
+
+var fetchWeek = function () {
+  $.ajax({
+    method: "GET",
+    url: "https://api.openweathermap.org/data/2.5/forecast?q=" + city.name + "&units=imperial&appid=" + config.apiKey,
+    dataType: "json",
+    success: function(data) {
+      addWeeksWeather(data);
+    },
+    error: function(jqXHR, textStatus, errorThrown) {
+      console.log(textStatus);
+    }
+  })
+}
+
+var addWeeksWeather = function (data) {
+  $('.five-day').empty();
+  var buildWeek = [];
+  for (let i = 7; i < data.list.length; i += 8) {
+    var wD = data.list[i];
+
+    var dayData = {
+      sky1: wD.weather[0].main,
+      temp1: Math.round(wD.main.temp),
+      imgURL1: wD.weather[0].icon,
+      day1: dayjs(wD.dt_txt).format('dddd'), 
+    }
+    buildWeek.push(dayData);
+  };
+
+  for (let j = 0; j < buildWeek.length; j++) {
```

Comment:
Try to avoid nested for loops if possible - this helps with simplicity and efficiency